bitnami/memcached/templates/service.yaml

```
@@ -21,6 +21,9 @@ spec:
   {{- if .Values.service.sessionAffinityConfig }}
   sessionAffinityConfig: {{- include "common.tplvalues.render" (dict "value" .Values.service.sessionAffinityConfig "context" $) | nindent 4 }}
   {{- end }}
+  {{- if not (empty .Values.service.trafficDistribution) }}
```

What do you think about this change? As it expects a string, it simplifies the code

```suggestion
  {{- if .Values.service.trafficDistribution }}
```